src/test/java/hw2/ex1/Ex1Test.java

```diff
@@ -0,0 +1,118 @@
+package hw2.ex1;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+import org.testng.asserts.SoftAssert;
+
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+
+public class Ex1Test {
+    WebDriver driver;
+    String url = "https://jdi-testing.github.io/jdi-light/index.html";
+    String login = "Roman";
+    String pass = "Jdi1234";
+    String user = "ROMAN IOVLEV";
+    String[] header = {"HOME", "CONTACT FORM", "SERVICE", "METALS & COLORS"};
+    String[] left = {"Home", "Contact form", "Service", "Metals & Colors", "Elements packs"};
+    String[] keyWord = new String[4];
+
+    @BeforeClass
+    public void setUp() {
+        driver = new ChromeDriver();
+        driver.manage().window().maximize();
+        driver.manage().timeouts().implicitlyWait(1, TimeUnit.SECONDS);
+        driver.manage().timeouts().pageLoadTimeout(3, TimeUnit.SECONDS);
+    }
+
+    @Test
+    public void ex1Test() {
+        SoftAssert softAssert = new SoftAssert();
+        //1. open site
+        driver.get(url);
+        softAssert.assertEquals(driver.getCurrentUrl(), url);
+
+        //2. Browser title equals "Home Page"
+        softAssert.assertEquals(driver.getTitle(), "Home Page");
+
+        //3. Perform login
+        driver.findElement(By.id("user-icon")).click();
+        driver.findElement(By.id("name")).sendKeys(login);
+        driver.findElement(By.id("password")).sendKeys(pass);
+        driver.findElement(By.id("login-button")).click();
+
+        //4. check user name on page after authorization
+        softAssert.assertEquals(driver.findElement(By.id("user-name")).getText(), user);
+
+        //5. Assert that there are 4 items on the header section are displayed and they have proper texts
+        List<WebElement> lsHeader = driver.findElements(By.cssSelector("ul.uui-navigation.nav.navbar-nav.m-l8 li"));
+        for (int i = 0; i < 3; i++) {
+            String actualText = lsHeader.get(i).getText();
+            softAssert.assertEquals(actualText, header[i]);  //5.1# check 1-3 items of header
+        }
+        String actText = driver.findElement(By.cssSelector("ul.uui-navigation.nav.navbar-nav.m-l8 [href='metals-colors.html']"))
+                .getText();
+        softAssert.assertEquals(actText, header[3]);  //5.2# check 4th items of header
+
+        //6. four images on Index page are displayed
+        List<WebElement> listBenefitIcons = driver.findElements
+                (By.cssSelector("div.row.clerafix.benefits span[class~=icons-benefit]"));
```

Comment:
in css classes can be referenced in any order:  a.first.second == a.second.first.
```suggestion
                (By.cssSelector("div.row.clerafix.benefits span.icons-benefit"));
```